server.py

```
@@ -6,39 +6,74 @@
 
 import os
 import json
+import logging
+import hashlib
 from datetime import datetime, timedelta
 from typing import Optional, Dict, List, Any, Tuple
 import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)